packages/cli/src/spec/mock/MockPrompt.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {Prompt} from '../../lib/Prompt';
+
+/**
+ * A an interface that promps for different types of user input.
+ */
+
+export class MockPrompt implements Prompt {
+  async printMessage(message: string): Promise<void> {
+    console.log(message);
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  async promptInput<T>(message: string): Promise<any> {
```

Comment:
I think this may want to return `Promise<T>`, the same way that `Prompt` does.

Reply:
I put there any on purpose, you can't return T type without using the validation function, and I don't need to use the validation function for this method purpose

Reply:
Ah OK. So I do think it would be good for this function to have the same signature as the function it is overriding on `Prompt`. There must be some way to create a `T`.

Reply:
I tried to think and read about it and didn't find anything

Reply:
I don't fully understand the implementation - MockPrompt is only being used right now in `createConfig`, right? That function always expects it to return a `string`. It seems the implementation could be simplified to the following:

```typescript
  async promptInput<T>(message: string): Promise<T> {
    return message as unknown as T;
  } 
```
Casting to `unknown` and then to `T` allows us to get rid of the eslint warning. An important thing is that, at this point, `MockLock` only works when prompting strings. If the expected object is different (eg: `Color` or `URL`) it won't work.

We should either make this more generic so it can be used anywhere or move the class to `ConfigSpec.ts`, where it is used. This PR is already quite large, so the last might be better for now.

Reply:
What does `as unknown` does?